### ISSUE TITLE
Google Pay - Update Wallet Version

### ISSUE THREAD

**Issue by nate-eisner** [braintree/braintree_android#446]
### General information

* SDK/Library version: 4.5.0
* Environment: Both
* Android Version and Device: All
* Braintree dependencies:
```
  com.braintreepayments.api:paypal
  com.braintreepayments.api:google-pay
  com.braintreepayments.api:card
```

### Issue description

I am working through the 3.x migration to 4.x and was wondering why we are required to use such an old version of Google wallet library? The latest version is com.google.android.gms:play-services-wallet:18.1.3 and you have it set for [16.0.1](https://github.com/braintree/braintree_android/blob/master/GooglePay/build.gradle#L38)

**Comment by sarahkoop** [comment]
Hi @nate-eisner - we can look into upgrading the wallet version, but in the meantime, feel free to open a PR with the requested change.

**Comment by sarahkoop** [comment]
This was released in `4.8.1`
